config.py

```diff
@@ -1,21 +1,48 @@
-class Config:
-    pass
+"""Docsstring for config.py."""
```

Comment:
Kindly remove the keyword `Docstring` in your comments since we already know that this is a docstring, they should be improved to be more descriptive documentations.